doc/manuals/admin/cli.md

```diff
@@ -143,3 +143,4 @@ The list of available options is the following:
 
 -   **-maxConnections**. maximum number of simultaneous connections. Default value is "unlimited" (limited by max file descriptors of operating system).
 -   **-reqPoolSize**. size of thread pool for incoming connections. Default value is 0, meaning *no thread pool*.
+-   **-reqTimeStat. turn on request-time-measuring in run-time. By default it is turned off (turned on, the broker will be somewhat slower in execution).
```

Comment:
Given that I'm going to improve documentation in this PR, I think I can take care of this fix.